src/DataSource.ts

```diff
@@ -1,28 +1,10 @@
-import { DataSourceInstanceSettings, MetricFindValue, DataQueryResponse, DataQueryRequest } from '@grafana/data';
-import { DataSourceWithBackend } from '@grafana/runtime';
-import { Observable } from 'rxjs';
-
+import { DataSourceInstanceSettings, MetricFindValue } from '@grafana/data';
+import { DataSourceWithBackend, getTemplateSrv } from '@grafana/runtime';
 import { InfluxQuery, InfluxOptions } from './types';
 
 export class DataSource extends DataSourceWithBackend<InfluxQuery, InfluxOptions> {
-  /** @ngInject */
-  instance: DataSourceInstanceSettings<InfluxOptions>;
-  constructor(instanceSettings: DataSourceInstanceSettings<InfluxOptions>, private templateSrv: any) {
```

Comment:
@srclosson FYI... if you make a constructor argument `private` it will become a member variable for the class.

Not needed... but worth pointin gut